components/aks/aks.tf

```
@@ -82,6 +82,17 @@ module "kubernetes" {
       node_taints         = []
       enable_auto_scaling = true
       mode                = "User"
+    },
+    {
+      name                = "cronjob"
+      vm_size             = lookup(var.cronjob, "vm_size", "Standard_DS3_v2")
```

set a sensible default
```suggestion
      vm_size             = lookup(var.cronjob, "vm_size", "Standard_D4ds_v5")
```